doc/building_sdk.md

```diff
@@ -1,34 +1,30 @@
-#### Building SDK
+## Building the SDK
 
 System Dependencies:
 
  - Mac OS X 10.8+ or Linux. kb-sdk does not run natively in Windows, but see [here](doc/FAQ.md#windows) for more details.
- - Java JRE 7 or 8 (9 is currently incompatible) http://www.oracle.com/technetwork/java/javase/downloads/index.html
+ - Java JRE 8: http://www.oracle.com/technetwork/java/javase/downloads/index.html (9 is currently incompatible; the SDK will run on Java 7, but using the more modern Java 8 is recommended)
  - (Mac only) Xcode https://developer.apple.com/xcode
  - git https://git-scm.com
  - Docker https://www.docker.com (for local testing)
 
 Get the SDK:
 
     git clone https://github.com/kbase/kb_sdk
+    git clone https://github.com/kbase/jars
 
 Pull dependencies and configure the SDK:
 
     cd kb_sdk
     make bin
 
-Download the local KBase SDK base Docker image:
-
-    docker pull kbase/sdkbase2:latest
-
```

Comment:
This is not the correct image for python apps, which would presumably be what the majority of people will be building. The correct image will be pulled automatically when building a new app, so it's easier to omit this step completely.

Reply:
So have we tried to rebuild some images with this setup? I'm somewhat leery when we remove a while directory.

Reply:
The automated tests for this repo all use this setup (i.e. ignoring the `sdkbase` directory), as does anyone who runs `docker pull kbase/kb-sdk`.

The only way that people might use the code is if they ran `make sdkbase`, which builds an app base image with the ID `kbase/kbase:sdkbase2.latest`. There were some vague/incomplete instructions in the docs about building `kb-sdk` from source about running `make sdkbase`, but I found that whenever I ran the command and then used `kb-sdk` to generate apps, the apps were on a weird base image, and some additional tests that I'd put in failed. I believe this was the precursor to the modern sdkbase images, which are now held in https://github.com/kbase/sdkbase2.